<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorMidiControlObserverForm</class>
 <widget class="QDialog" name="qtractorMidiControlObserverForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI Controller</string>
  </property>
  <property name="windowIcon">
   <iconset theme="itemControllers" />
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ControlTypeTextLabel">
     <property name="text">
      <string>&amp;Type:</string>
     </property>
     <property name="buddy">
      <cstring>ControlTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ControlTypeComboBox">
     <property name="toolTip">
      <string>MIDI event type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="ChannelTextLabel">
     <property name="text">
      <string>Cha&amp;nnel:</string>
     </property>
     <property name="buddy">
      <cstring>ChannelSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="ChannelSpinBox">
     <property name="toolTip">
      <string>MIDI channel</string>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="8">
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ParamTextLabel">
     <property name="text">
      <string>&amp;Parameter:</string>
     </property>
     <property name="buddy">
      <cstring>ParamComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="ParamComboBox">
     <property name="toolTip">
      <string>MIDI parameter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QCheckBox" name="LogarithmicCheckBox">
     <property name="text">
      <string>&amp;Logarithmic</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QCheckBox" name="FeedbackCheckBox">
     <property name="text">
      <string>&amp;Feedback</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QCheckBox" name="InvertCheckBox">
     <property name="text">
      <string>In&amp;vert</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QCheckBox" name="HookCheckBox">
     <property name="text">
      <string>&amp;Hook</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QCheckBox" name="LatchCheckBox">
     <property name="text">
      <string>L&amp;atch</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="InputsPushButton">
       <property name="toolTip">
        <string>Control input connections</string>
       </property>
       <property name="text">
        <string>&amp;Inputs</string>
       </property>
       <property name="icon">
        <iconset theme="itemMidiPortIn" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OutputsPushButton">
       <property name="toolTip">
        <string>Control output connections</string>
       </property>
       <property name="text">
        <string>&amp;Outputs</string>
       </property>
       <property name="icon">
        <iconset theme="itemMidiPortOut" />
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint">
        <size>
         <width>8</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="8"/>
 <tabstops>
  <tabstop>ControlTypeComboBox</tabstop>
  <tabstop>ChannelSpinBox</tabstop>
  <tabstop>ParamComboBox</tabstop>
  <tabstop>LogarithmicCheckBox</tabstop>
  <tabstop>FeedbackCheckBox</tabstop>
  <tabstop>InvertCheckBox</tabstop>
  <tabstop>HookCheckBox</tabstop>
  <tabstop>LatchCheckBox</tabstop>
  <tabstop>InputsPushButton</tabstop>
  <tabstop>OutputsPushButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc"/>
 </resources>
 <connections/>
</ui>
